gwf/cli/arg_parsing.py

```diff
@@ -0,0 +1,55 @@
+from __future__ import absolute_import, print_function
+from ..utils import cache, import_object
+from ..core import PreparedWorkflow
+import argparse
+
+
+class SubCommand(object):
```

Comment:
I think just `Command` is better than `SubCommand`. Otherwise `RunCommand` should be called `RunSubCommand`, but that's just ugly.